u/upath/upath_ubi_8.4.sh

```diff
@@ -0,0 +1,64 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : upath
+# Version       : v2.0.1
+# Source repo   : https://github.com/anodynos/upath
+# Tested on     : RHEL ubi 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : BulkPackageSearch Automation vikas.gupta8@ibm.com
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=upath
+PACKAGE_VERSION=v2.0.1
+PACKAGE_URL=https://github.com/anodynos/upath
+
+yum -y update && yum install -y yum-utils nodejs nodejs-devel nodejs-packaging npm python38 python38-devel ncurses git gcc gcc-c++ libffi libffi-devel ncurses git jq make cmake
```

Comment:
Please avoid `yum update` unless absolutely needed for package dependencies.

Reply:
Removed yum update.